document/core/syntax/instructions.rst

```diff
@@ -547,8 +547,10 @@ Instructions in this group are concerned with linear :ref:`memory <syntax-mem>`.
 
 .. math::
    \begin{array}{llcl}
+   \production{index type} & \X{it} &::=&
```

Comment:
```suggestion
   \production{index value} & \X{iv} &::=&
```
However, the grammar for u64 subsumes the one for u32, so you could just as well use u64 below. The binary grammar only uses u64 anyways. So we can just drop this.